tests/core/timeseries/test_io.py

```diff
@@ -582,6 +583,7 @@ def test_filter_by_date_interval(df_values):
 
 
 @pytest.mark.sql
+@pytest.mark.skipif(condition=sys.version_info[:2] == (3, 12), reason="Python 3.12 currently not supported by duckdb")
```

Comment:
Just a suggestion to write it a bit shorter. There is no need to slice the `version_info` tuple.
```suggestion
@pytest.mark.skipif(condition=sys.version_info < (3, 12), reason="DuckDB does not support Python 3.12 yet")
```

Reply:
Indeed condition needs to be `False` to make the test run, so we instead have to do 
```suggestion
@pytest.mark.skipif(not sys.version_info < (3, 12), reason="Python 3.12 currently not supported by duckdb")
```
but it still makes it more consise so I will change that!